src/platform/plugins/shared/data_views/server/rest_api_routes/public/create_data_view.ts

```diff
@@ -80,6 +80,7 @@ const registerCreateDataViewRouteFactory =
                 data_view: serviceKey === SERVICE_KEY ? dataViewSpecSchema : schema.never(),
                 index_pattern:
                   serviceKey === SERVICE_KEY_LEGACY ? dataViewSpecSchema : schema.never(),
+                managed: schema.maybe(schema.boolean({ defaultValue: false })),
```

Comment:
@davismcphee who would we need to update on the docs side about documenting this change in a release note? Not sure what the process is for platform here..

Reply:
I'm not too familiar with internal/external API, is removing this line sufficient to keep it internal?